### PR TITLE
Fix #1282: Add Support for fetching Kubernetes Metrics

### LINKED ISSUE #1282: [Question] How to get cpu and memory usage of a pod
How to write code to get cpu & memory usage of a pod, just like this command:
```
root@dfs0:~# kubectl top pod ojques-27stf
NAME           CPU(cores)   MEMORY(bytes)   
ojques-27stf   11m          426Mi
```
I searched the project, but can't find any file contains keyword `top`.
Thanks.

### PR DESCRIPTION
**Pull request by rohanKanojia**
Fix #1282 

This PR enables fetching metrics from Kubernetes server via these dsl 

```
   client.top().pod().metrics();
   client.top().node().metrics();
```

### CODE REVIEW

**Comment by oscerd** [comment]
Ok to test

File: kubernetes-model/kubernetes-model/src/main/java/io/fabric8/kubernetes/api/model/Duration.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+import io.sundr.builder.annotations.Inline;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.io.Serializable;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+/**
```
Comment (manusa) [inline kubernetes-model/kubernetes-model/src/main/java/io/fabric8/kubernetes/api/model/Duration.java]: Please, provide a class description or remove the comment

Comment (rohanKanojia) [reply kubernetes-model/kubernetes-model/src/main/java/io/fabric8/kubernetes/api/model/Duration.java]: oops :laughing:

**Comment by rohanKanojia** [comment]
[merge]

**Pull request merged**